viewer/src/index.ts

```diff
@@ -4,22 +4,19 @@
 
 // CAD types
 export { CadModel } from './models/cad/CadModel';
-export { CadNode } from './views/threejs/cad/CadNode';
+export { SectorMetadata } from './models/cad/types';
 export { CadRenderHints } from './views/CadRenderHints';
 export { CadLoadingHints } from './models/cad/CadLoadingHints';
-export { SectorNode } from './views/threejs/cad/SectorNode';
-export { SectorMetadata } from './models/cad/types';
 
 // Point cloud
-export { PointCloudModel } from './datasources/PointCloudModel';
+export { PointCloudModel } from './models/pointclouds/PointCloudModel';
 
 // Loaders
 export { createLocalCadModel, createLocalPointCloudModel } from './datasources/local';
-export { createThreeJsSectorNode, createThreeJsPointCloudNode } from './views/threejs';
-export { initializeCesiumSectorScene } from './views/cesiumjs/createCesiumSectorNode';
 
-// Utilities
-export { toThreeVector3 } from './views/threejs/utilities';
+// ThreeJS views
+import * as threejs from './views/threejs';
+export { threejs };
```

Comment:
Unfortunately, I think this style will make it impossible to have conditional imports in projects that depend on us later on. Now, I think three.js and cesium.js become hard, non-optional dependencies of our project. The reason it was in a different file was so that our users would be able to exclude three.js or cesium.js completely.

Reply:
But is that really a problem now? If we decide to split the project into 'base', 'threejs' and 'cesiumjs' - wouldn't the import statements and the usage be identical?